<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ticketsClass</class>
 <widget class="QMainWindow" name="ticketsClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>tickets</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(194, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="currentIndex">
         <number>4</number>
        </property>
        <widget class="QWidget" name="buyPage">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <item>
               <widget class="QLineEdit" name="lineEdit_4">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-style: solid; border-width: 2px; border-color: green;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>Откуда</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-style: solid; border-width: 2px; border-color: green;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>Куда</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Когда</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dateEdit_4">
                <property name="styleSheet">
                 <string notr="true">border-style: solid; border-width: 2px; border-color: green;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Искать маршруты с пересадкой</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 12pt; border-style: solid; border-width: 1px; border-color: green;</string>
              </property>
              <property name="text">
               <string>Найти билеты</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="refundPage">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>6000</width>
              <height>200</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(240, 244, 255); border-style: solid; border-width: 1px; border-color: green;</string>
            </property>
            <widget class="QPushButton" name="pushButton_4">
             <property name="geometry">
              <rect>
               <x>300</x>
               <y>30</y>
               <width>271</width>
               <height>71</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 16pt;</string>
             </property>
             <property name="text">
              <string>Оформить возврат</string>
             </property>
            </widget>
            <widget class="QWidget" name="verticalLayoutWidget_5">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>30</y>
               <width>291</width>
               <height>71</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Укажите уникальный номер билета:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_6">
                <property name="placeholderText">
                 <string>Введите ID</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="schedulePage">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="styleSheet">
             <string notr="true">font: 16pt;</string>
            </property>
            <property name="text">
             <string>Выберите дату:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCalendarWidget" name="calendarWidget_2">
            <property name="styleSheet">
             <string notr="true">color: black;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="styleSheet">
             <string notr="true">border-style: dotted; border-width:1px; border-color: blue;</string>
            </property>
            <property name="widgetResizable">
             <bool>false</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>600</width>
               <height>164</height>
              </rect>
             </property>
             <widget class="QWidget" name="horizontalLayoutWidget_10">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>50</y>
                <width>581</width>
                <height>31</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_24">
               <item>
                <widget class="QLabel" name="label_71">
                 <property name="text">
                  <string>Дата отбытия</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="dateEdit_46">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_72">
                 <property name="text">
                  <string>Дата прибытия</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="dateEdit_47">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_73">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Место отбытия-место прибытия; Рейс №</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="horizontalLayoutWidget_11">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>581</width>
                <height>31</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_25">
               <item>
                <widget class="QLabel" name="label_74">
                 <property name="text">
                  <string>Дата отбытия</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="dateEdit_48">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_75">
                 <property name="text">
                  <string>Дата прибытия</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="dateEdit_49">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_76">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Место отбытия-место прибытия; Рейс №</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="horizontalLayoutWidget_12">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>90</y>
                <width>581</width>
                <height>31</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_26">
               <item>
                <widget class="QLabel" name="label_77">
                 <property name="text">
                  <string>Дата отбытия</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="dateEdit_50">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_78">
                 <property name="text">
                  <string>Дата прибытия</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="dateEdit_51">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_79">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Место отбытия-место прибытия; Рейс №</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="horizontalLayoutWidget_14">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>120</y>
                <width>581</width>
                <height>31</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_28">
               <item>
                <widget class="QLabel" name="label_83">
                 <property name="text">
                  <string>Дата отбытия</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="dateEdit_54">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_84">
                 <property name="text">
                  <string>Дата прибытия</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="dateEdit_55">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_85">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Место отбытия-место прибытия; Рейс №</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="horizontalLayoutWidget_15">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>200</y>
                <width>581</width>
                <height>31</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_29">
               <item>
                <widget class="QLabel" name="label_86">
                 <property name="text">
                  <string>Дата отбытия</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="dateEdit_56">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_87">
                 <property name="text">
                  <string>Дата прибытия</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="dateEdit_57">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_88">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Место отбытия-место прибытия; Рейс №</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="FAQPage">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Часто задаваемые вопросы по приложению:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Вопросы и ответы&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="mainPage">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: white; border-style: solid; border-width: 1px; border-color: green;</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Добро пожаловать в наше приложение для покупки билетов на поезд!&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Вы можете купить билеты и посмотреть расписание, а также оформить возврат билета!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buyPageBtn">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>6000</width>
                <height>90</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: white; font: 12pt; border-style: solid; border-width: 1px; border-color: green;</string>
              </property>
              <property name="text">
               <string>Купить билеты</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Билеты</string>
    </property>
    <addaction name="buy"/>
    <addaction name="refund"/>
    <addaction name="schedule"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="FAQ"/>
    <addaction name="ask"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="personal"/>
    <addaction name="settings"/>
   </widget>
   <widget class="QMenu" name="MenuQuit">
    <property name="title">
     <string>Выход</string>
    </property>
    <addaction name="QuitMain"/>
    <addaction name="QuitAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
   <addaction name="MenuQuit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="buy">
   <property name="text">
    <string>Купить</string>
   </property>
  </action>
  <action name="refund">
   <property name="text">
    <string>Возврат</string>
   </property>
  </action>
  <action name="FAQ">
   <property name="text">
    <string>FAQ</string>
   </property>
  </action>
  <action name="ask">
   <property name="text">
    <string>Задать вопрос</string>
   </property>
  </action>
  <action name="personal">
   <property name="text">
    <string>Личные данные</string>
   </property>
  </action>
  <action name="settings">
   <property name="text">
    <string>Параметры приложения</string>
   </property>
  </action>
  <action name="schedule">
   <property name="text">
    <string>Расписание</string>
   </property>
  </action>
  <action name="QuitMain">
   <property name="text">
    <string>На главную страницу</string>
   </property>
  </action>
  <action name="QuitAction">
   <property name="text">
    <string>Выйти из приложения</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tickets.qrc"/>
 </resources>
 <connections/>
</ui>
